framework/Steps.py

```diff
@@ -433,6 +433,7 @@ def _localInitializeStep(self,inDictionary):
     #generate lambda function list to collect the output without checking the type
```

Comment:
No, because it's too late to stop RAVEN entirely once we get into the `ROM.evaluate` method, as it will count the individual runs as failed and keep going.  We have to check in the `Step` before running it.